st2actions/tests/unit/test_executions.py

```diff
@@ -72,7 +72,7 @@ def tearDown(self):
         super(TestActionExecutionHistoryWorker, self).tearDown()
 
     def test_basic_execution(self):
-        liveaction = LiveActionDB(action='core.local', parameters={'cmd': 'uname -a'})
+        liveaction = LiveActionDB(action='executions.local', parameters={'cmd': 'uname -a'})
```

Comment:
Yes, I know this is not the ideal name, but it's a smaller change which fixes the constraint violation mentioned in the PR description.

Eventually, we can move everything to `st2tests/fixtures/packs/core/`, but this requires more changes.